Add core-management-cliplugins and standalone-cliplugins packages (#1051)
vmware-tanzu/tanzu-framework: management-packages/standalone-cliplugins/README.md

# standalone-cliplugins Package

> This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not (and should not be) installed on the management cluster. Instead, it is used by the Tanzu CLI directly as an OCI bundle. In future, this package should be placed at more appropriate location.

Reviewer: also curious, where should this package live?

> This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not (and should not be) installed on the management cluster. Instead, it is used by the Tanzu CLI directly as an OCI bundle. In future, this package should be placed at more appropriate location.

Author: Yes. This might not be the exact place where we want to keep this package in the future and hence I have added the above comment as well as have added more details under the `Note` section in this same file.

However, is there a way we can publish a package and not add it as part of `PackageRepository` ?

> This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not (and should not be) installed on the management cluster. Instead, it is used by the Tanzu CLI directly as an OCI bundle. In future, this package should be placed at more appropriate location.

Author: As discussed in the meeting yesterday, refactoring was done as part of #1061 and this PR has been updated based on that to move this package into standalone packages.

## Components

* Required CLIPlugin resources for standalone plugins like `login`, `management-cluster`, `package` etc.

## Details

To learn more about the CLIPlugins and how it is getting used with Tanzu CLI, refer to this [doc](../../docs/design/context-aware-plugin-discovery-design.md)

## NOTE

> This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not directly getting installed on the management cluster but used by tanzu cli directly as an oci bundle. In future, this plugin should be placed at more appropriate location. 

Reviewer 2: **nitpick**
```suggestion
This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not (and should not be) installed on the management cluster. Instead, it is used by the Tanzu CLI directly as an OCI bundle. In future, this package should be placed at more appropriate location. 
```
